serveradmin/servershell/static/js/servershell.js

```diff
@@ -208,21 +208,21 @@ servershell.alert = function(text, level, auto_dismiss=5) {
         'dark'
     ];
 
-    if (!levels.includes(level))
+    if (!levels.includes(level)) {
         return;
+    }
 
-    let box = $('#js-alert');
-    levels.forEach(level => box.removeClass(`alert-${level}`));
-    box.addClass(`alert-${level}`);
-
-    let message = $('#js-alert-message');
-    message.text(text);
+    let template = $('.alert').first();
```

Comment:
Let's give that template an id. There should be just one template and id-search is faster than class search plus first.

Even better would be to have it as a template string inside js and then replace variables, but for that you either need an extra library or ugly string replacement. So, just leaving it as an information here, not for this PR.

Reply:
Using an id instead of an class would require to change/remove the id after cloning which imho reduces readibility and is also another function call. Since we are talking about microseconds here I am not sure if it is worth.

I don't like template strings in JS as the reduce readbility and make it hard for people not knowing the code to get a grasp of the page layout when looking at it. Do you think it has any advantages to make it a JS template string ?

Reply:
Clarified in the call. 

Encapsulating this in a JS libary for reusability and scalability is not necessary here in my opinion because the Servershell page is the only one using this functionality (client side notifications). Everything else should use the Django messaging framework (server side) and use the alert template from base.html. This should always be our first choice.